main/bitlbee/APKBUILD

```diff
@@ -2,28 +2,26 @@
 # Maintainer: Kiyoshi Aman <kiyoshi.aman@gmail.com>
 pkgname=bitlbee
 pkgver=3.5.1
-pkgrel=6
+pkgrel=7
 pkgdesc="An IRC to other chat networks gateway"
 url="https://www.bitlbee.org/"
 arch="all"
-license="GPL-2.0"
-makedepends="python3 glib-dev openssl-dev libotr-dev"
+license="GPL-2.0-or-later"
+makedepends="python3 glib-dev gnutls-dev libotr-dev"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-otr $pkgname-openrc"
 source="http://get.bitlbee.org/src/$pkgname-$pkgver.tar.gz
-	openssl-1.1.patch
 	$pkgname.initd
 	$pkgname.confd
 	"
 
-builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
 	cd "$builddir"
 	PYTHON=python3 \
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
-		--ssl=openssl \
+		--ssl=gnutls \
```

Comment:
Maybe it would make sense to a comment containing a link to https://bugs.bitlbee.org/ticket/886 to this line to make sure this isn't changed unless the upstream bug is resolved?

Reply:
Done, thank you